vendor/src/github.com/docker/notary/circle.yml

```diff
@@ -18,8 +18,6 @@ machine:
     CIRCLE_PAIN: "mode: set"
   # Put the coverage profile somewhere codecov's script can find it
     COVERPROFILE: coverage.out
```

Comment:
Because it cannot know whether or not they're used during compilation.  It
only explicitly removes "_test.go" files.